common/flatpak-run.c

```diff
@@ -424,6 +554,11 @@ flatpak_run_add_wayland_args (FlatpakBwrap *bwrap)
   gboolean res = FALSE;
   struct stat statbuf;
 
+#ifdef ENABLE_WAYLAND_SECURITY_CONTEXT
+  if (flatpak_run_add_wayland_security_context_args (bwrap, app_id))
```

Comment:
If the compositor supports the protocol and we still fail to create the context then shouldn't we error out here instead of giving the app unconstrained access?

I already don't know how the compositor would even know if flatpak is old or the app is from the host.

Reply:
Fixed.